.github/workflows/check.yml

```diff
@@ -90,6 +95,11 @@ jobs:
       env:
         MESON_OPTIONS: -Dinstalled-tests=true -Db_sanitize=${{ matrix.sanitizer }}
 
+    - name: Run dbus-daemon
+      run: |
+        $RUN_CMD $AS_USER sh -c "dbus-daemon --fork --session --print-address > dbus-env"
+        echo "DBUS_SESSION_BUS_ADDRESS=$($RUN_CMD $AS_USER cat dbus-env)" >> "$GITHUB_ENV"
```

Comment:
The pytest test suite should really run a dbus-daemon itself, either directly from Python code or [by telling Meson to wrap it in `dbus-run-session`](https://sources.debian.org/src/dbus-python/1.3.2-4/test/meson.build/?hl=72#L72). If it did that, then the Github workflow wouldn't need to, and that would also be nicer for running the pytest test suite on a developer's system or during rpm/dpkg/etc. build.

Reply:
good point, thanks. though staring at this again I realised that we're using `dbusmock` anyway which starts its own session bus for the test cases anyway. Not sure yet why this doesn't work in the container, need to debug this

Reply:
yup, works without, that hunk is dropped now. thanks again!